server/app/routes/utils.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+//exports
+module.exports = {
+  appPath,
+  dbPath,
+  bluebirdForTests,
+  createAdminUser,
+  createGuest,
+  createLocalUser,
+  Credentials,
+  db,
+  describeIt,
+  ensureAuthenticated,
+  _err,
+  forceSyncDb,
+  getApp,
+  getModel,
+  getModels,
+  joinRootPath,
+  logInAgent,
+  logOutAgent,
+  respondWith404,
+  responder,
+  routerUse,
+  Sequelize,
+  should,
+  supertest,
+  syncDb
+}
+
+
+
+// dependencies
+const path = require( 'path' );
+const bluebird = bluebirdForTests();
+
+//declarations
+const appPath = joinRootPath( "./server/app" );
+const dbPath = joinRootPath( "./server/db" );
+global.paths = {routerUtils: joinRootPath( './server/app/routes/utils.js')}
+
+function addUser( credentials ) {
+
+  // console.log(credentials);
+  return getModel('user').create( credentials )
+  .then( result => {
+    
+  })
+  .catch( err => console.log('oh no there was an error here!', err))
+}
+
+function bluebirdForTests() {
```

Comment:
I had considered adding a custom promise method to call .then(()=> done()).catch(done). 
I would have added it inside the bluebirdForTests method, so I could isolate the changed bluebird module from the rest of the app.

I decided not to, but left the method there in case we wanted to add any custom promise methods.